app/src/main/kotlin/io/homeassistant/companion/android/widgets/todo/TodoWidgetConfigureViewModel.kt

```diff
@@ -46,10 +49,11 @@ class TodoWidgetConfigureViewModel @Inject constructor(
             serverManager.integrationRepository(serverId)
                 .getEntities()
                 .orEmpty()
-                .filter { entity -> entity.domain == "todo" }
-        }.stateIn(viewModelScope, SharingStarted.WhileSubscribed(500), emptyList())
+                .filter { entity -> entity.domain == TODO_DOMAIN }
+        }.stateIn(viewModelScope, SharingStarted.WhileSubscribed(500.milliseconds), emptyList())
+
     var selectedEntityId by mutableStateOf<String?>(null)
-    var selectedBackgroundType by mutableStateOf(WidgetBackgroundType.DAYNIGHT)
+    var selectedBackgroundType by mutableStateOf(WidgetBackgroundType.DYNAMICCOLOR)
```

Comment:
Dynamic color is not available on all SDK versions, shouldn't there be a check for that?